src/types/user_operation.rs

```diff
@@ -45,20 +52,17 @@ impl From<UserOperation> for entry_point_api::UserOperation {
 
 impl UserOperation {
     pub fn pack(&self) -> Bytes {
-        self.clone().encode_hex().parse::<Bytes>().unwrap()
+        Bytes::from(self.clone().encode())
     }
 
     pub fn pack_for_signature(&self) -> Bytes {
-        let mut encoded = String::from("0x");
-        let packed = hex::encode(
-            UserOperation {
-                signature: Bytes::from_str("0x").unwrap(),
-                ..self.clone()
-            }
-            .encode(),
-        );
-        encoded.push_str(&packed[..packed.len() - 64]);
-        encoded.parse::<Bytes>().unwrap()
+        let mut packed: Vec<u8> = UserOperation {
+            signature: Bytes::default(),
+            ..self.clone()
+        }
+        .encode();
+        packed.truncate(packed.len() - 32);
+        Bytes::from(packed)
```

Comment:
We don't have to convert from bytes to str and then str to bytes which is unnecessary and slower. We could just operate on vec bytes.

Reply:
Looks good